### DISCUSSION TITLE
Showing Labels (dots) on specific dates

### DISCUSSION THREAD

**Discussion by shehzarr** [wojtekmaj/react-calendar#859] [Q&A] [answered]
Lets say I have events scheduled on a particular date. I want to show a dot (label) at the bottom of that date. 

Basically I am clicking on a date on the calendar and rendering my events on that date (onclick). 

so, i want some labeling system to be established on the bottom of dates so UI gives user the idea that a particular date has an event scheduled.

**Comment by samhirtarif** [comment, marked as answer]
@shehzarr - You could try using tileClassName like [this](https://stackblitz.com/edit/stackblitz-starters-nvy34p?file=src%2FApp.tsx,src%2Fstyle.css) to mark which dates are selected

If this helps, do mark this as the accepted answer 👍🏻
